compiler/lib/config.ml

```diff
@@ -118,6 +118,9 @@ module Param = struct
   let switch_max_case =
     p ~name:"switch_size" ~desc:"set the maximum number of case in a switch" (int 60)
 
+  let inlining_limit =
+    p ~name:"inlining-limit" ~desc:"set the size limit for inlining" (int 500)
+
```

Comment:
What unit is this in? Bytes?

Reply:
The unit is not really meaningful. What this feature compute is "number of instructions + number of branching"